.github/workflows/ci-pytorch_dockers.yml

```diff
@@ -123,6 +123,19 @@ jobs:
           push: false
         timeout-minutes: 95
 
+  build-NGC:
```

Comment:
Why do you need this now? Where was it getting built before if this didn't exist?

Reply:
it was build and published manually so we a few times lost compatibility